plugins/lsp/language-servers/default.nix

```
@@ -708,5 +713,6 @@ in
       ./pylsp.nix
       ./rust-analyzer.nix
       ./svelte.nix
+      tsserverRenameModule
```

I would either just do this inline:

```suggestion
      (lib.mkRenamedOptionModule (baseLspPath ++ [ "tsserver" ]) (baseLspPath ++ [ "ts-ls" ]))
```

Or I would have a set of renames we could expand on in the future:

```nix
  renamedServers = {
    tsserver = "ts-ls";
  };
  renameModules = mapAttrsToList (old: new: lib.mkRenamedOptionModule (baseLspPath ++ [ old ]) (baseLspPath ++ [ new ]) renamedServers
in
  lspModules
  ++ renameModules
  ++ [
    ./ccls.nix
    # ...
```